### ISSUE TITLE
Preview on merge function for imports

### ISSUE THREAD

**Issue by valeriosantarelli** [obdasystems/eddy#27]
It would be very helpful to have a preview of the number of conflicts that take place when importing a Graphol ontology in the current project.

Basically, before starting the merge process, the user should know how many conflicts there are, so he can decide if merging is reasonable or not.

**Comment by valeriosantarelli** [comment]
Stesso discorso della issue #26 , credo che la issue non sia più valida.
Aspetto conferma prima di chiudere con won't fix. @NonnonePalmiro
